scanomatic/io/jsonizer.py

```diff
@@ -245,3 +246,133 @@ def object_hook(obj: dict) -> Union[dict, Enum, Model]:
 
 def loads(s: Union[str, bytes]) -> Any:
     return json.loads(s, object_hook=object_hook)
+
```

Comment:
New implementations of things that previously was located in the abstract factory file.